server/src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -419,106 +414,14 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         }
 
         // meta data
+        // TODO MetaData.toXContent
         if (metrics.contains(Metric.METADATA)) {
-            builder.startObject("metadata");
-            builder.field("cluster_uuid", metaData().clusterUUID());
-
-            builder.startObject("cluster_coordination");
-            coordinationMetaData().toXContent(builder, params);
-            builder.endObject();
-
-            builder.startObject("templates");
```

Comment:
Would be good to have an example that shows how / whether templates are rendered differently now.

Reply:
I have it here as an intermediate commit https://github.com/elastic/elasticsearch/pull/48924/commits/bfc1103163f4cb1fb6c49d5a3bff06e50a30eef6 . Please let me know if this look good.